src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = '';
+
+  if (days >= 7) {
+    result = days * 40 - 50;
+  } else if (days >= 3) {
+    result = days * 40 - 20;
+  } else {
+    result = days * 40;
```

Comment:
Don't duplicate yourself. Create a variable for `days * 40` to make your code cleaner.
Same with all main values (40, 20, 50 etc.)